css-two-column-layout/styles.css

```diff
@@ -0,0 +1,77 @@
+body {
+  font-family: 'Karla', sans-serif;
+}
+
+.container {
+  width: 85%;
+  margin: auto;
+  text-align: center;
+}
+
+.header {
+  width: 65%;
+  margin: auto;
+  padding-top: 40px;
+}
+
+h1 {
+  letter-spacing: 4px;
+  font-size: 2.8rem;
+  margin-bottom: 15px;
+}
+
+h4 {
+  letter-spacing: 1px;
+  font-size: 0.8rem;
+  margin-top: 30px;
+  margin-bottom: 10px;
+}
+
+p {
+  font-size: 0.8rem;
+  color: gray;
+  font-weight: 500;
+  line-height: 21px;
+  margin-top: 5px;
+}
+
+.row-full {
+  display: flex;
+}
+
+.column-one-half {
+  text-align: center;
+  margin: 20px;
+  margin-bottom: 24px;
+}
+
+.column-one-fourth {
+  text-align: left;
+  width: 60%;
+}
+
+.row-one-half {
+  display: flex;
+}
```

Comment:
This does the same thing as `row-full`. You should delete this, and rename `row-full` to `row`